public/stylesheets/weekly-review.css

```diff
@@ -0,0 +1,86 @@
+.weekly-summary-modal {
+  display: none;
+  font-family: 'Open Sans', Arial;
+  background-color: white;
+  width: 370px;
+  height: 367px;
+  position:absolute;
+  left: 50%;
+  top: 50%;
+  margin-left: -200px;
+  margin-top: -200px;
+  z-index: 101;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
```

Comment:
Please add spaces between opening and closing parentheses, and arguments, in a function invocation or control flow expression:
```js
fn(param1, param2) { /* ... */ }
if(expression) { /* ... */ }
```
should be
```js
fn( params1, param2 ) { /* ... */ }
if( expression ) { /* ... */ }
```